integration-tests/mongodb/src/test/java/org/apache/camel/quarkus/component/mongodb/it/MongoDbTest.java

```diff
@@ -277,13 +280,19 @@ public void testOutputTypeDocument() throws Exception {
     }
 
     private void waitAndResetTailingResults(int expectedSize, String laststring, String resultId) {
-        await().atMost(5, TimeUnit.SECONDS).until(
-                () -> RestAssured
-                        .given().contentType(ContentType.JSON)
-                        .get("/mongodb/results/" + resultId)
-                        .then()
-                        .statusCode(200)
-                        .extract().as(Map.class),
+        LOG.info("Awaiting results with size " + expectedSize + " and last string " + laststring);
+        await().atMost(30, TimeUnit.SECONDS).until(
```

Comment:
I hope this is enough for CI build. I used 20 seconds on my machine yesterday and it was still failing.

Reply:
Let's see if the timeout helps, and also let's see what the debug output will be when it fails next time.

Reply:
@zbendhiba If it failed with 20 seconds on your machine, was it successful with some higher timeout?
(I remember, that it was able to finish on my machine in less then a few seconds and if I remember correctly,  Peter and Alex were running it ~10 times both and it was successful)